webworker/package-validation.ts

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
name files after functions, put each function in a file named after the function